Makefile

```diff
@@ -1,5 +1,5 @@
-DOCKER_REGISTRY_COMPONENTS := the_registry_to_use_for_components
-DOCKER_REGISTRY_SECRET := your_already_installed_secrets
+DOCKER_REGISTRY_COMPONENTS := res-drl-hpc-docker-local.artifactory.swg-devops.com
+DOCKER_REGISTRY_SECRET := art-drl-hpc
```

Comment:
please revert these changes

Reply:
Fixed this in 9c66ac156452e7d4c3fd366a85116342b694b03f